src/Glpi/OAuth/Server.php

```diff
@@ -226,31 +232,33 @@ private static function doGenerateKeys(): void
         ];
 
         // Generate key
-        $key = openssl_pkey_new($config);
-        if ($key === false) {
-            $error = openssl_error_string();
-            throw new RuntimeException("Unable to generate keys: $error");
+        try {
+            $key = openssl_pkey_new($config);
+        } catch (OpensslException $e) {
+            throw new RuntimeException("Unable to generate keys: " . $e->getMessage());
         }
 
         // Export private key to file
-        if (!openssl_pkey_export_to_file($key, self::PRIVATE_KEY_PATH)) {
-            $error = openssl_error_string();
-            throw new RuntimeException("Unable to export private key: $error");
+        try {
+            openssl_pkey_export_to_file($key, self::PRIVATE_KEY_PATH);
+        } catch (OpensslException $e) {
+            throw new RuntimeException("Unable to export private key: " . $e->getMessage());
         }
 
         // Get public key
-        $pubkey = openssl_pkey_get_details($key);
+        $pubkey = openssl_pkey_get_details($key); // @phpstan-ignore-line Safe\openssl_pkey_get_details only exists for PHP >= 8.4
```

Comment:
This will probably trigger an error on PHP 8.2 beacause PHPStan will fail to ignore an error that does not exist. This is why the PHP 8.4 specific baseline was required.

Reply:
Yes, unfortunately... I put it back; I see no other solution